syntax = "proto2";

package tutorial;

message OrderRecord
{
    required string id = 1;
    required int32 currency_pair = 2;
    required int32 direction = 3;
    required int32 status = 4;
    required int64 datetime = 5;
    required double init_px = 6;
    required double init_volume = 7;
    required double fill_px = 8;
    required double fill_volume = 9;
    required string desc = 10;
    required string tags = 11;
    required int32 zone = 12;
    required double currency_price = 13;
}